index.html

```diff
@@ -1913,6 +1919,67 @@
 
 </aside>
 
+## Transparency
+
+<div class="practice">
+  <span class="practicelab" id="transparency-when-requested">
+    When accessing data or requesting permission, [=sites=] (and other [=actors=]) should provide
+    [=people=] with relevant information about its use and [=user agents=] should help present and
+    consume that information.
+  </span>
+</div>
+
+Transparency is a necessary, but insufficient, condition for [=consent=]. Relevant information
+includes who is accessing information, what information is accessed (including the potential
+inferences or combinations of such data) and how information is used. For transparency to be
+meaningful to people, information must be provided in the relevant [=context=].
```

Comment:
It's not completely clear whether the "information" that must be provided in context is 1) the information that gets accessed or 2) the "relevant information" that helps users make informed choices. Contexts are used in the rest of the document to provide principles about information use, which leans toward the first interpretation, but I think you mean the second.

Reply:
Clarified with "explanatory information".